charts/spire/charts/spire-server/templates/controller-manager-webhook.yaml

```diff
@@ -10,7 +10,7 @@ webhooks:
         name: {{ include "spire-controller-manager.fullname" . }}-webhook
         namespace: {{ include "spire-server.namespace" . }}
         path: /validate-spire-spiffe-io-v1alpha1-clusterfederatedtrustdomain
-    failurePolicy: {{ .Values.controllerManager.validatingWebhookConfiguration.failurePolicy }}
+    failurePolicy: Ignore # Actual value to be set by post install/upgrade hooks
```

Comment:
Will need to set  the other webhook to ignore in this file too.

Reply:
Should be covering both now. Did I miss something?

Reply:
Oh. I think I had an older version of the file. This is looking ok to me now.